### PR TITLE
Upgraded spring boot

### PR DESCRIPTION
**Pull request by Jefex7**
### JIRA link (if applicable) ###

[https://tools.hmcts.net/jira/browse/EM-5115](url)

### Change description ###

Spring boot 3 upgrade

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -12,7 +12,7 @@ properties([
     ])
 ])
 
-@Library("Infrastructure")
+@Library("Infrastructure@enable-logging-on-ft")
```
Comment (mdayican) [inline Jenkinsfile_CNP:15]: pls revert back

Comment (Jefex7) [reply Jenkinsfile_CNP:15]: Okay

**Review by mdayican — APPROVED** [review]

**Review by MichaelJacob21012 — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -116,22 +110,19 @@ dependencies {
...
     implementation group:'com.fasterxml.jackson.core', name: 'jackson-databind',version: versions.fasterXmlJackson
     implementation group:'com.fasterxml.jackson.core', name: 'jackson-core', version: versions.fasterXmlJackson
     implementation group:'com.fasterxml.jackson.module', name: 'jackson-module-afterburner', version: versions.fasterXmlJackson
 
-    implementation group: 'javax.el', name: 'javax.el-api', version: '3.0.0'
-    implementation group: 'org.zalando', name: 'problem-spring-web', version: versions.zalando
-    implementation group: 'org.zalando', name: 'jackson-datatype-problem', version: versions.zalando
+    implementation group: 'org.zalando', name: 'problem-spring-web', version: '0.29.1'
+    implementation group: 'org.zalando', name: 'jackson-datatype-problem', version: '0.27.1'
     implementation group: 'org.springdoc', name: 'springdoc-openapi-ui', version: '1.7.0'
```
Comment (mdayican) [inline build.gradle:126]: `implementation group: 'org.springdoc', name: 'springdoc-openapi-starter-webmvc-ui', version: '2.1.0'`

you need to change'springdoc' to swagger work with spring3

Comment (Jefex7) [reply build.gradle:126]: Okay I will do the change now

**Review by yogesh-hullatti — APPROVED** [review]

**Pull request merged**